src/js/jsx/sections/style/Shadow.jsx

```diff
@@ -471,32 +475,23 @@ define(function (require, exports) {
                     );
                 }, this).toList();
             } else {
-                shadowsContent = (
+                shadowsContent = Immutable.List.of((
                     <div className="shadow-list__list-container__mixed">
                         <i>{strings.STYLE.DROP_SHADOW.MIXED}</i>
                     </div>
-                );
+                ));
             }
 
-            // we may want to gate the add dropshadow button to PS's max amout of drop shadows.
+            if (shadowsContent.isEmpty()) {
+                return null;
+            }
 
             return (
-                <div className="shadow-list__container">
-                    <header className="shadow-list__header sub-header">
-                        <h3>
+                <div className="shadow-list__container ">
+                    <header className="shadow-list__header section-header section-header__no-padding">
```

Comment:
nit: this smells like it should be simpler.  you can safely ignore this comment.  There's probably a reason.